relayer/pkg/chainlink/relay.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/smartcontractkit/chainlink-starknet/relayer/pkg/chainlink/ocr2"
-	"github.com/smartcontractkit/chainlink-starknet/relayer/pkg/starknet"
+	chainset "github.com/smartcontractkit/chainlink-starknet/relayer/pkg/starknet/chain"
```

Comment:
I don't have a better suggestion, but `chainset` seems too specific as the whole packge alias.

Reply:
`starkchain` ?

Reply:
`ChainSet` is a Chainlink concept, not a StarkNet concept.

IMHO the `ChainSet` type should be moved form `pkg/starknet` to `pkg/chainlink`.
I'd suggest we leave `client.go`, `client_test.go` and `types.go` in `pkg/starknet` and move everything else to `pkg/chainlink` or `pkg/chainlink/ocr2` depending on domain the code handles.